src/components/Alphabet/Letter/Letter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './Letter.css';
+
+const checkLetter = (letter, id, clickedLetter) => {
+    const url = `http://localhost:3000/phrases/check?id=${id}&letter=${letter}`
+    return fetch(url, {
+        method: 'post',
+    })
+    .then(response => response.json())
+    .catch(error => {
+        console.error(error);
+    })
+    .then(data => {
+        clickedLetter(data);
+    });
+}
+
+const Letter = ({letter, gameID, clickedLetter}) => {
+    console.log("gameID",gameID)
+    return(
+        <div className={styles.letter} onClick={checkLetter.bind(null, letter, gameID, clickedLetter)}>
```

Comment:
this binding is kind of unnecessary - it would be much better to convert Letter into React.Component and have an internal method, that takes those 2 values from props and calls checkLetter inside, so:
```
someMethod = () => {
    const { checkLetter, letter, gameID } = this.props;
    checkLetter(letter, gameID);
}
...
render () {
...
    onClick={this.someMethod}
}
```

Reply:
Ok, I will do that.